pyannote/audio/models/blocks/pooling.py

```diff
@@ -47,27 +47,29 @@ def forward(
 
         Parameters
         ----------
-        sequences : (batch, channel, frames) torch.Tensor
+        sequences : (batch, channel, frames) or (batch, speakers, channel, frame) torch.Tensor
             Sequences.
-        weights : (batch, frames) torch.Tensor, optional
+        weights : (batch, frames) or (batch, speakers, frames) torch.Tensor, optional
```

Comment:
Actually, what we need to support is the following combinations of shapes:

sequences  | weights     | output
-----------|-------------|-------------
`(b, c, F)`| `(b, f)`    | `(b, c)`
`(b, c, F)`| `(b, s, f)` | `(b, s, c)`

A `sequences` of shape `(b, c, s, F)` will never happen in practice.

Also, not the different values of `F` and `f`.